src/components/success_rate/index.js

```diff
@@ -10,14 +10,22 @@ const SuccessRate = ({success,failed})=> {
                 </div>
                 :
                 <>
-                    {(success >= failed)?
+                    {(failed == 0)?
                         <div class={style['green-bar']}>
                             
                         </div>
                         :
-                        <div class={style['red-bar']}>
+                        <>
+                            {(success == 0)?
+                                <div class={style['red-bar']}>
+                                    
+                                </div>
```

Comment:
```suggestion
                                <div class={style['red-bar']} />
```
ini coba diimprove kayak gini kayaknya bisa fer